agent/resources.go

```diff
@@ -0,0 +1,25 @@
+package agent
+
+import ()
+
+type Resources struct {
+	CpuTotal    uint64 `json:cpu_total`
+	CpuUsage    uint64 `json:cpu_usage`
+	MemoryTotal uint64 `json:memory_total`
+	MemoryUsage uint64 `json:memory_usage`
+	DiskTotal   uint64 `json:disk_total`
+	DiskUsage   uint64 `json:disk_usage`
+}
```

Comment:
we design the resource as the interface, so it's easy for us to extend the system, we can easily write other resource